test/e2e/inspect_test.go

```diff
@@ -524,10 +524,12 @@ var _ = Describe("Podman inspect", func() {
 
 		data := inspect.InspectContainerToJSON()
 		Expect(data).To(HaveLen(1))
-		Expect(data[0].HostConfig.CapDrop).To(HaveLen(3))
+		Expect(data[0].HostConfig.CapDrop).To(HaveLen(4))
 		Expect(data[0].HostConfig.CapDrop[0]).To(Equal("CAP_AUDIT_WRITE"))
 		Expect(data[0].HostConfig.CapDrop[1]).To(Equal("CAP_MKNOD"))
 		Expect(data[0].HostConfig.CapDrop[2]).To(Equal("CAP_NET_RAW"))
+		// FIXME: Why is sys_chroot dropped?
+		Expect(data[0].HostConfig.CapDrop[3]).To(Equal("CAP_SYS_CHROOT"))
```

Comment:
@rhatdan PTAL

Reply:
A bug in computing the dropped caps?  Need to figure that out before 4.4.

Reply:
This was changed in https://github.com/containers/common/pull/1240

Reply:
But why are they listed as dropped?  There are over 30 capabilities and other ones we don't enable by default aren't listed as dropped.

Reply:
Ah I see what you mean, looks like inspect uses not the container.conf default but rather the oci runtime spec default:
https://github.com/containers/podman/blob/34cc61d167fe429eb76383cf2a09701a9b2533dd/libpod/container_inspect_linux.go#L129-L177